src/lib/components/Map/Map.js

```diff
@@ -184,12 +186,27 @@ class Map extends Component {
     const feature = features[0];
     if (!feature) return false;
     const activeLayerObj = this.props.layersObj.find((l) => l.id === feature.layer.id);
+    /**
+     * Todo:
+     * Investigate adjusting zoom to show selected feature
+     */
+    if (feature.layer && feature.layer.id === 'nutrition-sites-live') {
+      /**
+       * 1. Set Icon opacity to one for the highlight layer
+       * 2. Filter out the selected feature
+       * 3. Filter in highlight layer feature with simillar facility_id
+       */
+      this.map.setFilter(NUTRITION_SITES_LIVE, [ALL, ["!=", FACILITY_ID, feature.properties.facility_id]]);
```

Comment:
Instead of using layer id constants, we can just use the `activeLayerObj.id` because if you refactor the cond above to what I suggested you will still get the nutritions sites id from activeLayerObj. Also, facility id can be found in `activeLayerObj.source.join[0]`